CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.4.1] - 2017-12-17
+### Added
+- `no_std` support
+
```

Comment:
A few other changes:
- Added HC-128 RNG. Maybe with the description that it is similar to ISAAC, but proven to be cryptographically secure and often faster.
- Technically `Isaac64Rng::next_u32` is a breaking change, because it no longer throws away half the results.
- `{IsaacRng, Isaac64Rng}::new_unseeded` is no longer available. Use `{IsaacRng, Isaac64Rng}::new_from_u64(0)` as replacement.
- RNG's no longer implement `Copy`.
- RNG's have a custom Debug implementation that does not expose the internal state.

Does it make sense to say something about `impls`? Or not advertise it too widely, because it may move to `rand_core`?

Reply:
I don't think any of that's merged yet.

Reply:
I thought it was your goal to include them in this release after https://github.com/rust-lang-nursery/rand/issues/205#issuecomment-352272614. Because they contain (small) breaking changes, they can't be part of 0.4.* if we follow semver.

Reply:
Thanks for the comments. Didn't think about lack of `Copy` being a breaking change, but you're right, it is (maybe we should add a deprecated impl?). `new_unseeded` we can potentially add back as a `deprecated` wrapper fn. So perhaps those changes can't land in 0.4 after all, or at least not all of them :/

BTW it should have been clear from this PR what was/wasn't in the release.

Reply:
I think I finally get the scheme you have in mind 😄